api/rpc/stack/stack.go

```diff
@@ -119,7 +126,9 @@ func (s *Server) processService(ctx context.Context, stack *Stack, serv *service
 	request := &service.ServiceCreateRequest{
 		ServiceSpec: serv,
 	}
-	server := service.Service{}
+	server := service.Service{
```

Comment:
I removed the direct function Create and Remove to let only the Service *Create and Service *Remove
Because the direct functions (Create and Remove) need to use a Docker client as a local variable  (*client.Client) and this variable does't exist anymore. It is accessible only in the Service struct. 
So Stack use a Service struct to invoke Service *Create and *Remove, but it's as efficient as the direct function call.